docs/book/src/developers/development.md

```diff
@@ -286,7 +286,7 @@ export CONTROL_PLANE_MACHINE_COUNT=3
 export AZURE_CONTROL_PLANE_MACHINE_TYPE="Standard_D2s_v3"
 export AZURE_NODE_MACHINE_TYPE="Standard_D2s_v3"
 export WORKER_MACHINE_COUNT=2
-export KUBERNETES_VERSION="v1.19.7"
+export KUBERNETES_VERSION="v1.21.2"
```

Comment:
given that there seems to be a bug in cloud provider (https://github.com/kubernetes-sigs/cloud-provider-azure/issues/706) that only affects Windows and fixing it will require cherry picks which will take a while, should we consider adding a temporary k8s version var for that specific test to keep it at 1.19. so we can at least get some coverage on 1.21 in the meantime?

@jsturtevant wdyt?

Reply:
makes sense to me

Reply:
Do the prior patch releases for 1.21 work? could we pin it to an earlier version in 1.21 for windows?

Reply:
I tried with 1.20.8 and had the same issue

Reply:
we think the bug was introduced in https://github.com/kubernetes/kubernetes/commit/667b2be621de91a4254af6499ce742332c981b76 which is in 1.20.0-beta.2 +